_schemaV2.graphql

```diff
@@ -17717,23 +17723,18 @@ type RecentlySoldArtworkTypeEdge {
   node: RecentlySoldArtworkType
 }
 
-# Autogenerated input type of RecordArtworkView
-input RecordArtworkViewInput {
```

Comment:
Did a cool thingy @mzikherman taught me - if not sure whether schema will add a breaking change - copy new schema to clients and run `yarn relay`. I did it in Force and Eigen. Force doesn't complain, but Eigen does:

```json
[ERROR] Error: ✖︎ Unknown type 'RecordArtworkViewInput'. Did you mean `RecordArtworkViewMutationInput`?

  src/app/Scenes/Artwork/Artwork.tsx:261:59
  260 │
  261 │         mutation ArtworkMarkAsRecentlyViewedQuery($input: RecordArtworkViewInput!) {
      │                                                           ^^^^^^^^^^^^^^^^^^^^^^
  262 │           recordArtworkView(input: $input) {

[ERROR] Compilation failed.
```

Reply:
Thanks!! Yeah, this will cause a compilation error for sure because Eigen is explicitly referencing `RecordArtworkViewInput` which has been removed in this diff. I wonder though if this would also cause a runtime error? Will check on that... if this doesn't cause any runtime backwards-incompatibility, I think we could merge this in (with type name changes) and follow-up with an Eigen PR to address the compilation error.

Reply:
Don't quote me on this - but I _think_ changing the name like that, if a Relay client explicitly references the name - will indeed cause a runtime error if you merge the schema update. Give it a whirl to confirm (and we start to increase our mental model of how to think about these kinds of changes with our clients).